.travis.yml

```diff
@@ -1,17 +1,70 @@
 # we use both C and C++, so advertize C++
 language: cpp
 cache: ccache
-dist: trusty
-compiler:
-  - gcc
```

Comment:
If this is removed, which compiler is being used for the `env` part?

Reply:
Looks like it defaults to gcc https://travis-ci.org/geany/geany/jobs/526317392#L442